File: py/IEC62325/InfIEC62325/InfDomain/EnergyTypeCode.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106 on Thu Dec 28 19:52:13 2023
from enum import Enum

class EnergyTypeCode(Enum):
    DASE = 1  # Day Ahead Scheduled Energy
    DSSE = 2  # Day Ahead Incremental Self Schedule Energy
    DABE = 3  # Day Ahead Incremental Energy Bid Awarded Energy
    OE = 4    # Optimal Energy
    HASE = 5  # Hour ahead pre-dispatched schedule energy
    SRE = 6   # Standard Ramping Energy
    RED = 7   # Ramping Energy Deviation
    EDE = 8   # Exceptional Dispatch energy
    RMRE = 9  # RMR Energy
    MSSLFE = 10  # MSSLF Energy
    RE = 11   # Residual Energy
    MLE = 12  # Minimum Load Energy
    SE = 13   # SLIC Energy
    RTSSE = 14  # Real time self scheduled energy
    DMLE = 15  # Day ahead minimum load energy
    PE = 16   # Pumping Energy
    TEE = 17  # Total Expected Energy
    DAPE = 18  # Day-Ahead Pumping Energy

File: java/IEC62325/InfIEC62325/InfDomain/EnergyTypeCode.java
package IEC62325.InfIEC62325.InfDomain;


/**
 * Valid Enumerations:
 * 1) DASE Day Ahead Scheduled Energy;
 * 2) DSSE Day Ahead Incremental Self Schedule Energy;
 * 3) DABE Day Ahead Incremental Energy Bid Awarded Energy;
 * 4) OE Optimal Energy;
 * 5) HASE Hour ahead pre-dispatched schedule energy;
 * 6) SRE Standard Ramping Energy;
 * 7) RED Ramping Energy Deviation;
 * 8) EDE Exceptional Dispatch energy;
 * 9) RMRE RMR Energy;
 * 10) MSSLFE MSSLF Energy;
 * 11) RE Residual Energy;
 * 12) MLE Minimum Load Energy;
 * 13) SE SLIC Energy;
 * 14) RTSSE Real time self scheduled energy;
 * 15) DMLE Day ahead minimum load energy;
 * 16) PE Pumping Energy;
 * 17) TEE Total Expected Energy;
 * 18) DAPE - Day-Ahead Pumping Energy;
 * @created 03-Jan-2024 1:48:21 PM
 */
public enum EnergyTypeCode {
	DASE,
	DSSE,
	DABE,
	OE,
	HASE,
	SRE,
	RED,
	EDE,
	RMRE,
	MSSLFE,
	RE,
	MLE,
	SE,
	RTSSE,
	DMLE,
	PE,
	TEE,
	DAPE
}